packages/grafana-ui/src/components/DateTimePickers/TimeRangePicker.tsx

```diff
@@ -156,6 +156,8 @@ export function UnthemedTimeRangePicker(props: TimeRangePickerProps): ReactEleme
   );
 }
 
+UnthemedTimeRangePicker.displayName = 'UnthemedTimeRangePicker';
```

Comment:
Maybe I'm missing something but I'm under the impression this is inferred from the name of the function or class. 🤔

Reply:
this is a really weird one that i still haven't fully got to the bottom of... basically, if you remove this display name, storybook doesn't infer the controls correctly? 😕 it's also why the story is using `UnthemedTimeRangePicker` instead of the plain `TimeRangePicker`.

i suspect it's because it's using the (deprecated) `withTheme` wrapper, and converting to `withTheme2` etc would help. @kaydelaney has got some separate PRs where she's doing that, so i was planning to revisit after they've been delivered to see if  this could be removed again 😅 or if you know how to fix it otherwise, point me in the correct direction and i'll do it 👍

Reply:
update: merged main to pick up kay's changes, and it still doesn't generate the storybook controls unless a `displayName` is set 😬